src/main/java/com/example/library/dto/BookSearchParametersDto.java

```diff
@@ -0,0 +1,4 @@
+package com.example.library.dto;
+
+public record BookSearchParametersDto(String title, String author, String isbn) {

```

Comment:
Let's add the ability to search by multiple values ​​of one filter parameter
```suggestion
public record BookSearchParametersDto(String title[], String author[], String isbn[]) {

```